appserver/ejb/ejb-full-container/src/main/java/org/glassfish/ejb/mdb/deployment/annotation/handlers/MessageDrivenHandler.java

```diff
@@ -114,7 +117,7 @@ protected EjbDescriptor createEjbDescriptor(String elementName,
      * Set Annotation information to Descriptor.
      * This method will also be invoked for an existing descriptor with
      * annotation as user may not specific a complete xml.
-     * @param ejbDesc
+     * @param ejbDescMaxWaitTimeInMillis
```

Comment:
?
The param name _is_ `ejbDesc`

Reply:
Good catch, it's also not used for "MaxWaitTimeInMillis", so doesn't seem like a matter of forgetting to rename the method argument as well.